test/standalone/emit_asm_and_bin/.gitignore

```diff
@@ -0,0 +1,2 @@
+main
+main.s
```

Comment:
please don't add these to .gitignore. this garbage should not be output to these directories in the first place.